client/src/charts/canada/canada.js

```diff
@@ -200,8 +200,8 @@ export class Canada extends React.Component {
       })
     );
     return (
-      <div className="frow no-container">
-        <div className="fcol-md-3">
+      <div className="row no-container">
```

Comment:
Pretty sure `.no-container` doesn't exist anymore, it was from `flexbox-grid.scss`. Did you double check the classes tossed out with the rest of `flexbox-grid.scss` and clean them up as needed?

Reply:
Whoops, I think this was the only one I accidentally skipped haha

I did check all the other one's during the first commit, so I doubt there's another one that managed to sneak past me.
The only one i would imagine that got past is `reverse` since that one is used all over the project with different contexts (ex: the reverse method). But I triple checked the find function on VSCode, so I'm pretty sure there was none